incidents/2022-12-27.md

```diff
@@ -0,0 +1,25 @@
+# December 27, 2022 SDK Preview doesn't work
+
+|                          |               |
+| -----------------------: | :------------ |
+| **Reported on Slack**          | Dec-27-2022   |
+|           **Mitigation** | Dec-27-2022   |
+|   **Solution Completed** | Dec-28-2022   |
+
+## What happened?
+
+After updating all peers from Catalyst v5.1.10 to v5.1.12 the nodes stopped serving the base wearables (like the `BaseFemale`) on any available endpoint for this need. This lead to an issue on the SDK preview where the users could not use this feature at all and also affected the avatar preview and edition on both SDK and Explorer. That made that the users were not able to edit or deploy a new version of their avatar.
```

Comment:
this lead => this led
where the users => where users
That made that the users were not able to edit... => That made it impossible for users to be able to edit...